### DISCUSSION TITLE
Whisper API for Base64 encoded webm audio

### DISCUSSION THREAD

**Discussion by snmishra311** [openai/whisper#1125] [Q&A]
Hi,
I'm using Retool to create a frontend where I'm recording an audio clip and want to transcribe this audio using Whisper. The audio is in the format Base64 encoded webm. I have tried decoding the audio to create a Buffer in NodeJS and pass it into my POST request but it fails

Would really appreciate any help on how to pass the encoded audio into Whisper correctly

**Comment by jpreethi1522** [comment]
even i am facing with the same issue.did you resolve it??
